lib/semian/mysql2.rb

```diff
@@ -26,9 +26,6 @@ module Mysql2
       /Too many connections/i,
       /closed MySQL connection/i,
       /MySQL client is not connected/i,
-    )
-
-    TIMEOUT_ERROR = Regexp.union(
```

Comment:
Is this public API ?

Reply:
Doubt so.

I mean it's accessible since there's no `private_constant`, but it's not documented or anything.